include/updater/csv_source.hpp

```diff
@@ -0,0 +1,19 @@
+#ifndef OSRM_UPDATER_CSV_SOURCE_HPP
+#define OSRM_UPDATER_CSV_SOURCE_HPP
+
+#include "updater/source.hpp"
+
+namespace osrm
+{
+namespace updater
+{
+namespace csv
+{
+    SegmentLookupTable readSegmentValues(const std::vector<std::string> &paths);
+    TurnLookupTable readTurnValues(const std::vector<std::string> &paths);
+}
```

Comment:
Nice this allows us to easily rip out csvs and implement something else.

_hint hint_ @karenzshea